### DISCUSSION TITLE
Improve TM with respect of text with xml tags

### DISCUSSION THREAD

**Discussion by ohallot** [WeblateOrg/weblate#11908] [General]
Hi

I'm translating docs with tons of XML. Some strings are clean (no XML) and others have XML intermixed. weblate translation memory fails to offer a suggestion in the following example case: 

"Hello World!" 

and 

"&lt;mytag a="1234"&gt;Hello Word!&lt;/mytag&gt;. 

The first string is not offered as suggestion to the (later) second string.

Is there a setting to put in TM so it can ignore/bypass xml tags?

Thank you.

**Comment by nijel** [comment]
This sounds like#11179 which was fixed in d53bce1bcda0b95940bd00fa045ea10017d0cde6.

**Reply by ohallot** [reply]
Bug #11179 seems to address machine translation issue. 
Here, my issue is internal translation memory used in weblate. Machine translation is not available in the installation I use.

**Reply by nijel** [reply]
I see. There is currently no such option and Weblate suggests only very similar strings (using [pg_trgm](https://www.postgresql.org/docs/current/pgtrgm.html)).
